test/index.js

```diff
@@ -26,8 +26,11 @@ describe('default resolution', () => {
       subscription: new GraphQLObjectType({
         name: 'Subscription',
         fields: {
-          foo: subscriptionWithClientId({
+          foo: subscription({
             name: 'FooSubscription',
+            inputFields: {
+              dummy: { type: GraphQLString },
```

Comment:
It's this check in graphql-js: https://github.com/graphql/graphql-js/blob/a819fb3a575f27d077a460f9a4327610b87abe40/src/type/definition.js#L911-L916

And then https://github.com/facebook/relay/blob/master/scripts/babel-relay-plugin/lib/RelayQLPrinter.js#L272 -> https://github.com/facebook/relay/blob/master/scripts/babel-relay-plugin/lib/RelayQLPrinter.js#L643-L645 in babel-relay-plugin that enforce that subscriptions take a single `input` argument. Though I guess the default could be a nullable string or something.